cosmwasm/cli-utils.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+require('dotenv').config();
+
+const { addExtendedOptions } = require('../common');
+const { governanceAddress } = require('./utils');
+
+const { Option } = require('commander');
+
+const addCommonAmplifierOptions = (program, options = {}) => {
+    const ops = {
+        ignoreParallel: true,
+        ignoreSaveChainSeparately: true,
+        ignoreGasOptions: true,
+        ignoreChainNames: true,
+        ignorePrivateKey: true,
+        ignoreVerify: true,
+        contractName: true,
+        salt: true,
+        ...options,
+    };
+
+    addExtendedOptions(program, ops);
```

Comment:
i'd say just use baseOptions, extendedOptions is more evm specific. Create `addDeployOptions` perhaps that adds extra params like salt, instantiate2, artifactPath etc.

Reply:
Since we refactored the utils to add `addEnvOptions`, just define the other options directly instead of using addBaseOptions/addExtendedOptions